corehq/apps/auditcare/couch_to_sql.py

```diff
@@ -54,12 +54,37 @@ def _get_couch_docs(start_key, end_key):
     return list(result)
 
 
+def get_unsaved_events(nav_objs, access_objs, nav_couch_ids, access_couch_ids):
```

Comment:
nit: I like the [newspaper metaphore](https://github.com/jbarroso/clean-code#the-newspaper-metaphor) as a way of structuring modules since it makes the code easier to read top-to-bottom. What do you think of moving this function below `get_events_from_couch`?